release-notes/2.1/2.1.1-known-issues.md

```diff
@@ -55,10 +55,10 @@ This issue will be fixed in the first update for 2.1.
 
 Using `dotnet install` or `dotnet uninstall` to manage Global Tools sometimes results in an `access denied` error. This occurs because another process, like an indexer, has not released a handle on the Global Tools files.
 
-*Workaround* 
+### Workaround
 
 **Global Tools created with .NET Core SDK 2.1.300 Preview 1 are not compatible with later versions.**
 
 Also: Ensure that global.json does not pin to .NET Core SDK 2.1.300 Preview 1 on either for a  directory where you are developing global tools or a directory from which you are installing global tools.
 
-*Workaround* - Manually remove these tools.
\ No newline at end of file
+*Workaround* - Manually remove these tools.
```

Comment:
should this also be formatted the same way as the previous workaround section changes?

Reply:
Probably. Yet these differences are all over the place. Maybe for a new pr? If needed ..

Reply:
Sounds good.